news/2024/2024-08-12-the-followpoint-motion-the-ultimate-catcher.md

```diff
@@ -0,0 +1,294 @@
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3885626">Motion</a></p>
+
+Back in January 2014, one of my friends introduced me to osu!. When I gave it a try, it immediately reminded me of Elite Beat Agents from the Nintendo DS Lite that I used to play a lot in my childhood. It was really fun to play in terms of similar style, gameplay to Elite Beat Agents, and a lot of different genres and songs.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Why did you choose to play osu!catch of all game modes osu! has to offer? And was it your first game mode? Since you are also really good at osu!mania.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3885626" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-Motion.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3885626">Motion</a></p>
+
+Actually, my first game mode was neither osu!catch nor osu!mania. But one of the main reasons why I tried and eventually fell in love with osu!catch was [ExGon](https://osu.ppy.sh/users/214187) who had been dominating for a long time - who is still the greatest of all time for us.
+
+After watching his plays on both osu! and youtube, I naturally tried that game mode and it ended up being my main game mode.
+As for osu!mania, I will go into detail in a later question, but I was already playing a game called O2jam before I started to play osu!.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Following up on my previous question, one of community members wants to know what's your secret to being so good at osu!mania, while not having much playtime in it?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3885626" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-Motion.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3885626">Motion</a></p>
+
+As I already mentioned in the previous question, I've played O2jam since 2006 so I was already used to playing VSRG-style games like O2jam and osu!mania. As a result I was able to adapt quickly to osu!mania without much playtime.
```

Comment:
```suggestion
As I already mentioned in the previous question, I've played O2Jam since 2006 so I was already used to playing VSRG-style games like O2Jam and osu!mania. As a result I was able to adapt quickly to osu!mania without much playtime.
```